versions/3.1.0.md

```diff
@@ -2321,7 +2321,7 @@ The following properties are taken from the JSON Schema definition but their def
 
 Alternatively, any time a Schema Object can be used, a [Reference Object](#referenceObject) can be used in its place. This allows referencing definitions instead of defining them inline.
 
-In addition to the JSON Schema properties, the following OpenAPI vocabulary properties MAY be used for further schema documentation:
+In addition to the JSON Schema properties defined in the JSON Schema Core and JSON Schema Validation vocabularies, any properties can be used from any vocabularies, or entirely arbitrary keywords. OpenAPI has its own vocabulary properties which MAY be used for further schema description:
```

Comment:
@philsturgeon 

> JSON Schema Core and JSON Schema Validation vocabularies

This should probably be one of the following (changes italicized):

* JSON Schema Core and JSON Schema Validation _specifications_
* _vocabularies defined in the_ JSON Schema Core and JSON Schema Validation _specifications_
* _Core, Applicator, Validation, Meta-Data, Format, and Content vocabularies from the_  JSON Schema Core and JSON Schema Validation _specifications_

The Core and Validation specs each define multiple vocabularies, one of which matches the spec name 😕

In place of

> OpenAPI has its own vocabulary properties

maybe

* The OpenAPI Specification defines an additional vocabulary of keywords which MAY be used along with the JSON Schema vocabulary keywords

not sure that's great, but I want to try to align the usage of the "vocabulary" concept.  It will fit better with how this will show up in the OAS 3.1 schema itself.

Reply:
That's a good point. My preference would be to go with the second suggestion. It makes the precidence clear without information overload.

Most people probably won't be interested to know about the specific vocabularies from the JSON Schema spec, although you could argue that those reading this section are likely to be such.

Reply:
Sorted, thanks!